packages/components/app/styles/components/tag.scss

```
@@ -0,0 +1,92 @@
+//
+// TAG COMPONENT
+//
+// properties within each class are sorted alphabetically
+//
+//
+
+$hds-tag-border-radius: 13px;
+
+.hds-tag {
+  align-items: stretch;
+  background-color: var(--token-color-surface-interactive);
+  border: 1px solid var(--token-color-border-strong);
+  border-radius: $hds-tag-border-radius;
+  color: var(--token-color-foreground-primary);
+  display: inline-flex;
+  font-family: var(--token-typography-font-stack-text);
+  font-size: 0.8125rem; // 13px
+  font-weight: var(--token-typography-font-weight-medium);
+  line-height: 1rem; // 16px
+  vertical-align: middle;
+}
+
+.hds-tag__dismiss {
+  flex: 0 0 auto;
+  border-radius: inherit;
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
```

I _think_ you can do this, if you want to reduce the "visual noise":
```suggestion
  border-radius: inherit 0 0 inherit;
```

---

Individual subproperties don't inherit, unfortunately, so we'll have to use the overrides on certain corners

---

Noooo way!! Didn't know this is a "special case". TIL today 🤷‍♂️